src/components/card/card.test.js

```diff
@@ -0,0 +1,400 @@
+          width: "260px",
+          display: "flex",
+          justifyContent: "center",
+          flexWrap: "wrap",
+        }}
+        m={2}
+        data-element={dataElement}
+      >
+        <Typography variant="p" color={getTextColor()}>
+          {title}
+        </Typography>
+        {children}
+      </Box>
+    );
+  };
+  const [items, setItems] = useState(cards);
+  const moveCardHandler = (dragIndex, hoverIndex) => {
+    const dragItem = items[dragIndex];
+    if (dragItem) {
+      setItems((prevState) => {
+        const copiedStateArray = [...prevState];
+        const prevItem = copiedStateArray.splice(hoverIndex, 1, dragItem);
+        copiedStateArray.splice(dragIndex, 1, prevItem[0]);
+        return copiedStateArray;
+      });
+    }
+  };
+  const returnItemsForColumn = (columnName) => {
+    return items
+      .filter((item) => item.column === columnName)
+      .map((item, index) => (
+        <MovableItem
+          key={item.id}
+          name={item.name}
+          currentColumnName={item.column}
+          setItems={setItems}
+          index={index}
+          moveCardHandler={moveCardHandler}
+          dataElement={`draggable-card-${item.id}`}
+        />
+      ));
+  };
+  const { productOne, productTwo } = columnNames;
+  return (
+    <Box width="700px" height="450px">
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "space-around",
+        }}
+      >
+        <DndProvider backend={HTML5Backend}>
+          <Column title={productOne} dataElement="draggable-container-1">
+            {returnItemsForColumn(productOne)}
+          </Column>
+          <Column title={productTwo} dataElement="draggable-container-2">
+            {returnItemsForColumn(productTwo)}
+          </Column>
+        </DndProvider>
+      </div>
+    </Box>
+  );
+};
+
+const CardTextAlignment = ({ ...props }) => {
+  return (
+    <Card>
+      <CardRow>
+        <CardColumn align="left">
+          <Heading title="Stripe - [account name]" divider={false} />
+          <Typography variant="h5">user.name@sage.com</Typography>
+        </CardColumn>
+        <CardColumn align="right">
+          <Icon type="image" />
+        </CardColumn>
+      </CardRow>
+      <CardRow pt={2} pb={2}>
+        <CardColumn {...props}>
+          <Typography variant="h5" weight="bold">
+            Stripe Balance
+          </Typography>
+          <Heading title="£ 0.00" divider={false} />
+          <Typography>LAST ENTRY: 15 DAYS AGO</Typography>
+        </CardColumn>
+      </CardRow>
+      <CardFooter>
+        <CardColumn>
+          <Link icon="link" href="https://carbon.sage.com/">
+            View Stripe Dashboard
+          </Link>
+        </CardColumn>
+      </CardFooter>
+    </Card>
+  );
+};
+
+context("Tests for Card component", () => {
+  describe("Should check Card component properties", () => {
+    it.each([
+      ["small", "24px"],
```

Comment:
you can use here a `number`? as:
```suggestion
      ["small", 24],
```